design/account.go

```diff
@@ -203,6 +203,7 @@ var userDataAttributes = a.Type("UserDataAttributes", func() {
 	a.Attribute("username", d.String, "The username")
 	a.Attribute("registrationCompleted", d.Boolean, "Whether the registration has been completed")
 	a.Attribute("email", d.String, "The email")
+	a.Attribute("email_private", d.Boolean, "Whether the email address would be visible to other users.")
```

Comment:
email_private=true means not visible,
email_private=false means visible, right?
With that logic the proper description should be "Whether the email address wouldn't be visible to other users" or  "Whether the email address would be private"